tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -836,6 +837,92 @@ def fully_connected(inputs,
 
 
 @add_arg_scope
+def layer_norm(inputs,
+               center=True,
+               scale=False,
+               epsilon=0.001,
+               activation_fn=None,
+               reuse=None,
```

Comment:
You're right, I wrote that before I noticed the absence of update_ops and saw what you were doing, sorry for the noise.